pkg/cmd/tknpac/bootstrap/kubestuff.go

```diff
@@ -44,9 +46,24 @@ func checkSecret(ctx context.Context, run *params.Run, opts *bootstrapOpts) bool
 }
 
 // check if we have the namespace created
-func checkNS(ctx context.Context, run *params.Run, opts *bootstrapOpts) (bool, error) {
-	ns, err := run.Clients.Kube.CoreV1().Namespaces().Get(ctx, opts.targetNamespace, metav1.GetOptions{})
-	return ns.GetName() != "", err
+func checkNS(ctx context.Context, run *params.Run, targetNamespace string) (bool, error) {
+	ns, err := run.Clients.Kube.CoreV1().Namespaces().Get(ctx, targetNamespace, metav1.GetOptions{})
+	if err != nil {
+		return false, err
```

Comment:
it doesn't return not found error?  🤔

Reply:
kube client-go would do that for us :D

Reply:
we have `ns.GetName() == "" ` this check so was confused when this would be the case 😄

Reply:
yeah i read that and i am not sure why i was doing it to  be honest :D 

i tested all the permutations manually as well...